packages/mui-codemod/src/deprecations/speed-dial-props/test-cases/theme.expected.js

```diff
@@ -0,0 +1,37 @@
+fn({
+  MuiSpeedDial: {
+    defaultProps: {
+      TransitionComponentProps: CustomTransitionProps,
```

Comment:
This should be 

```
slotProps: {
    transition: CustomTransitionProps,
}
```

I wondered why the test wasn't failing, and it's because we're missing the test file, for example: https://github.com/mui/material-ui/blob/next/packages/mui-codemod/src/deprecations/backdrop-props/backdrop-props.test.js

Reply:
forgive me for my insolence....test file added 😅